code/modules/admin/admin_verbs.dm

```diff
@@ -636,9 +637,12 @@ GLOBAL_LIST_INIT(admin_verbs_maintainer, list(
 
 	log_admin("[key_name(usr)] deadmined themself.")
 	message_admins("[key_name_admin(usr)] deadmined themself.")
+	if(check_rights(R_ADMIN|R_MOD, FALSE))
+		GLOB.de_admins.Add(ckey)
```

Comment:
Also use +=

I know it looks bad but it's more performance because it's a native operation not a proc jump.